tests/lib/util/comment-event-generator.js

```diff
@@ -32,7 +32,7 @@ describe("NodeEventGenerator", function() {
     it("should generate comment events without duplicate.", function() {
         var emitter = new EventEmitter();
         var generator = new NodeEventGenerator(emitter);
-        var code = "//foo\nvar zzz /*aaa*/ = 777\n//bar";
+        var code = "//foo\nvar zzz /*aaa*/ = 777;\n//bar";
```

Comment:
This code example was broken due to ASI differences in Espree 2.x. In 3.x, there are no differences so updated just to make it clear.